### DISCUSSION TITLE
Welcome to atsresume Discussions!

### DISCUSSION THREAD

**Discussion by sauravhathi** [sauravhathi/atsresume#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by 1md3nd** [comment]
Hey, 
I was quiet impressed by your project. I have few skills which will be very helpful to your project. 
Can I contribute to your project and make it better.
Thanks

**Reply by sauravhathi** [reply]
Thank you! We appreciate your interest. This is open source, anyone can contribute.
